tin-drink/src/components/content/drinkList/DrinkList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import SingleDrink from './SingleDrink'
+
+class DrinkList extends React.Component {
+
+    state = {
+        drinks: []
+    }
+
+    test = () => {
+        console.log('arbert')
+    }
+
+    test = () => {
+        var x =8;
+        console.log(ddf);
+
+        (func)
```

Comment:
Expected an assignment or function call and instead saw an expression.
'func' is not defined.